ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -268,12 +276,24 @@ struct MSFListCellView: View, ConfigurableTokenizedControl {
     }
 
     var body: some View {
+        let labelColor: Color
+        let sublabelColor: Color
+        let trailingItemColor: DynamicColor
+        if state.isSelected {
+            labelColor = Color(dynamicColor: tokens.labelSelectedColor)
```

Comment:
I assume it's "because that's what the figma defines", but is there a reason the label and sublabel colors aren't dynamic?

Reply:
The label color didn't really need to change, but we were calling `Color(dynamicColor: tokens.sublabelColor)` twice, so I just changed them to be `Color` here instead. `DynamicColor` is mostly an internal class we use to hold multiple different related colors, but if we want to actually use it in the UI we need to change it to either `Color` (for Swift) or `UIColor` (for UIKit (on iOS, Mac would need NSColor, but we don't do that yet)).

Reply:
Though, I guess technically, `DynamicColor` is a public struct, so it's neither internal nor a class. But, what I meant was it's something we added to group colors together.